pre_commit_dbt/check_exposure_has_models.py

```diff
@@ -0,0 +1,34 @@
+from pre_commit_dbt.utils import get_filenames, get_exposures
+from typing import Sequence
+from pathlib import Path
+from typing import Optional
+import argparse
+from pre_commit_dbt.utils import add_filenames_args
+import logging
+
+def has_models(paths: Sequence[str]):
+    status_code = 0
+    ymls = get_filenames(paths, [".yml", ".yaml"])
+    exposures = get_exposures(list(ymls.values()))
+    missing_model = { exposure.exposure_name for exposure in exposures if exposure.models is None}
+    for exposure in missing_model:
+        status_code = 1
+        print(
```

Comment:
I had the same question. If print is what they were using it can be fine but as in these hooks we could like to specify if it's a warning, an error or something else I think logs would be way better.